tests/unit/_modules/test_osd.py

```diff
@@ -717,7 +717,7 @@ def test_bluestore_partitions_wal_and_db_log(self, mock_log):
         Expect to call log() ( and leave the partition creation to ceph-disk )
         """
         kwargs = {'format':
-                  'bluestore',
+                      'bluestore',
```

Comment:
I think this is indented too far :thinking:

Reply:
true..